libnetdata/queue/queue.c

```diff
@@ -0,0 +1,85 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+
+#include "queue.h"
+
+struct nodes{
```

Comment:
The plural is confusing this is just a single node, ie. `struct node`.

Reply:
Actually "s" is not the plural form but states "structs" and differentiates it from type definition.